src/main/java/com/atlassian/bitbucket/jenkins/internal/model/BitbucketDefaultBranch.java

```diff
@@ -15,11 +15,10 @@ public class BitbucketDefaultBranch extends BitbucketRef {
 
     @JsonCreator
     public BitbucketDefaultBranch(@JsonProperty("id") String id, @JsonProperty("displayId") String displayId,
-                                  @JsonProperty("type") BitbucketRefType type,
                                   @JsonProperty("latestCommit") String latestCommit,
                                   @JsonProperty("latestChangeset") String latestChangeset,
                                   @JsonProperty("isDefault") boolean isDefault) {
-        super(id, displayId, type);
```

Comment:
why this change?
Are we not getting the correct `type` coming through?

Reply:
It's been a bit since i've looked at this. I'll double check

Reply:
This may have been a temporary change when debugging some tag issues. Reverted to previous implementation.